src/backend/input.rs

```diff
@@ -216,6 +231,9 @@ pub trait InputBackend: Sized {
     /// Get current `InputConfig`
     fn input_config(&mut self) -> &mut Self::InputConfig;
 
+    /// Called to inform the Input backend about a new focused Output for a `Seat`
+    fn set_output_metadata(&mut self, seat: &Seat, output: &Output);
```

Comment:
Do we expect the input backend to need more information than just the dimensions of the output currently focused?

Also, we could go an other route: as the result of "transformed" values from libinput are floats, could we just use `(width, height) = (1,1)` and define the result from the backend to be exprimed as a ration value relative to the total size of the output, and let the handler deal with that ?